nautobot/utilities/testing/views.py

```diff
@@ -85,7 +85,7 @@ def model_to_dict(self, instance, fields, api=False):
 
             if api:
 
-                # Replace ContentType numeric IDs with <app_label>.<model>
+                # Replace ContentType UUIDs with <app_label>.<model>
```

Comment:
This is one that needs to stay as numeric IDs, because `ContentType` is a built-in model that we cannot modify to use UUID PKs. Just change it to "primary keys" and it'll be accurate in any context.